Source/Core/InputCommon/ControllerEmu/ControlGroup/ControlGroup.h

```diff
@@ -48,11 +48,19 @@ enum class GroupType
 class ControlGroup
 {
 public:
-  explicit ControlGroup(std::string name, GroupType type = GroupType::Other);
-  ControlGroup(std::string name, std::string ui_name, GroupType type = GroupType::Other);
+  enum CanBeDisabledOption
```

Comment:
I think this should really be an `enum class`. Unless you can think of a better name, how about this?
```c++
enum class CanBeDisabled
{
  Yes,
  No,
};
```